test/libsolidity/semanticTests/multiSource/external/external.sol

```diff
@@ -0,0 +1,11 @@
+// Exclude From Tests
```

Comment:
I think we may need to ensure that external sources can import too, such as `import "./token.sol";`.

How complicated would that be? Also we do not necessarily want to allow actual test cases in externalsources. The goal would be we include externalsources, and then use them in the importing source to build test cases, etc.

Reply:
Ah thats true!